dashboard/app/assets/stylesheets/application.scss

```diff
@@ -4269,3 +4269,48 @@ a {
 .cm-editor {
   height: 100%;
 }
+
+#signup {
```

Comment:
I know you don't want nits for UI but curious if this should be in a scss module instead? (If you agree but don't want to update now, a TODO would be fine imo)

Reply:
I was considering this and am happy to- I think it would make sense once we set up the partial that will take care of all the webpack loading so it isn't just one more file I'm pulling into the whole application😂

Reply:
Made a ticket for it here https://codedotorg.atlassian.net/browse/ACQ-2183

Reply:
Sorry I don't think I was clear -- I was imagining a SCSS module in our apps directory such as https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/certificates/congrats.module.scss

Reply:
(So wouldn't need a shared partial for that piece specifically)

Reply:
Either way, still happy for this to be merged without that change. Just wasn't sure if a different ticket needed to be filed? Or if I'm missing something of course!